GUIDELINES.md

```diff
@@ -38,3 +38,15 @@ Use an English phrase instead. This is not a man page.
 ## Use backticks around libcurl constants
 
 For example, `CURLE_OK` and `CURLOPT_URL`.
+
+## Consistent spelling
+
+Use American English spellings.  Use the following spellings of these terms
+that could be spelled multiple ways;
+
+* back-end
+* e-mail
```

Comment:
Out of this list, I think "e-mail" is the one that I have the harder to type myself. It seems that more sources also agree that both forms are "correct" so maybe we can just go with the dash-less version?

https://www.grammarly.com/blog/spelling-e-mail-email/ - looks like a decent overview

Reply:
In the current text:

email: 37 instances
e-mail: 10 instances

Reply:
For email I think both works. I am not sure what is meant by the setup entry. All the other words look like typos to me and I would write them like: backend, runtime, toolchain and wildcard.